src/view/DogList.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import Dogs from '../components/Dogs/Dogs';
+import { getDogs } from '../services/dogs';
+import './DogList.css';
+
+export default function DogList() {
+  const [dogs, setDogs] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await getDogs();
+      setDogs(data);
+    };
+    fetchData();
+    setLoading(false);
+  }, []);
+
+  if (loading) return <h1>LOADING</h1>;
+
+  return (
+    <div className="dog-list">
+      {dogs.map((d) => (
```

Comment:
Suggestion: 'dog' over 'd' for better readability

Reply:
I actually find this way to be more easily readable for myself, especially with that variable only being used in that one map. Were I using it in a place with a broader scope, I would use a more descriptive name. I just find it less likely for me to get confused by formatting my maps in this way.